components/svg/close_icon.jsx

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {FormattedMessage} from 'react-intl';
+
+export default class CloseIcon extends React.PureComponent {
+    render() {
+        return (
+            <span {...this.props}>
+                <FormattedMessage
+                    id='generic_icons.close'
+                    defaultMessage='Close Icon'
+                >
+                    {(ariaLabel) => (
+                        <svg
+                            width='24px'
+                            height='24px'
+                            viewBox='0 0 24 24'
+                            role='icon'
+                            aria-label={ariaLabel}
+                        >
+                            <path d='M19,6.41L17.59,5L12,10.59L6.41,5L5,6.41L10.59,12L5,17.59L6.41,19L12,13.41L17.59,19L19,17.59L13.41,12L19,6.41Z'/>
+                        </svg>
+                    )}
+                </FormattedMessage>
```

Comment:
I don't entirely understand this, but am I right in thinking this results in the formatted message only appearing in the Aria Label variable?

Reply:
This is another way to call formatted message allowing to pass its children as a render function, this is copy from the rest of the SVGs components. I'm just coping the pattern here.

Reply:
That's cool - I didn't know about that.

Reply:
I only found out about that fairly recently as well. I would've highlighted it in my "down with `Utils.localizeMessage`" talk a while back if I had known about it then 😛